app/src/main/java/org/mozilla/fenix/exceptions/ExceptionsView.kt

```diff
@@ -21,21 +20,30 @@ abstract class ExceptionsView<T : Any>(
     protected val interactor: ExceptionsInteractor<T>
 ) : LayoutContainer {
```

Comment:
nit: I think we don't need `LayoutContainer` as we are using view binding :)

Reply:
I blame all of this on @Mugurell

Reply:
Right. This is a `android.view.View/ViewGroup`but not a `kotlinx.android.extensions.LayoutContainer`.

Reply:
We'll have to make sure that when the migration ends there is also no `kotlinx.android.extensions` import anywhere in the project.

Reply:
Maybe I'm missing something :), but I think the import is from  `kotlinx.android.extensions.LayoutContainer`

![image](https://user-images.githubusercontent.com/773158/129211946-e116b097-a631-434c-89c6-db8f31c2f414.png)

Reply:
@Amejia481 Agree. And when the migration ends there should't be any trace of this import in the project. Sorry, I think we were overlapping in comments.
I was saying that this is a `android.view.ViewGroup` (which receives the parent layout in which it inflates and attaches itself) and can be declared as such but is not a `kotlinx.android.extensions.LayoutContainer` so this is a something that should be refactored as part of this migration work.

Reply:
Removing `LayoutContainer` breaks `TrackingProtectionExceptionsView`. Unsure what to do next.

Reply:
Same changes I here, just making the `binding` property from `ExceptionsView` protected to be available in `TrackingProtectionExceptionsView` seems to work for me.

Reply:
Then I'll have to wait for https://github.com/mozilla-mobile/fenix/pull/20788/ to merge